### DISCUSSION TITLE
why doesn't GITHUB support respond in a timely manor

### DISCUSSION THREAD

**Discussion by rsj-github** [community/community#66003] [New to GitHub]
I've had to setup a new account recently as i mucked up the old one and the 2fa associated to it. the old account has my work email now set to primary and i'm not able to remove this as i have no access to the old account.
what i wanted to know is does the support tickets actually work as i've had zero response from github support.

Is there an actual support team ???

**Comment by queenofcorgis** [comment]
Hi @rsj-github, 

Thank you for your patience. The community is unable to assist with nor escalate your ticket, but we can assure you it is in the right place. It will be answered in the order it was received based on our team's resources and the amount of tickets ahead of yours. We appreciate your understanding! 

As no other Community Discussions team members will be able to provide additional help with account related questions I am going to close this discussion.

Thank you!

**Reply by rsj-github** [reply]
its ok i've managed to sort it out myself
